browser/brave_profile_prefs.cc

```diff
@@ -437,6 +438,8 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->SetDefaultPrefValue(policy::policy_prefs::kEnableDirectSockets,
                                 base::Value(false));
 
+  BraveFarblingService::RegisterProfilePrefs(registry);
```

Comment:
is this indented correctly? Looks weird, but might just be github

Reply:
entire PR passes lint checks at the moment